universe/jupyterlab/1/config.json

```diff
@@ -259,6 +259,16 @@
           "type": "string",
           "default": "spark.mesos.containerizer=mesos"
         },
+        "spark_conf_mesos_principal": {
+          "description": "Spark Mesos Principal.",
```

Comment:
Did you want to add more elaborate name/descriptions from https://spark.apache.org/docs/latest/running-on-mesos.html ?

Or is that another PR?

Reply:
Hmmn... good suggestion. I could add it just here. I finished this PR before I started working on the refactor PR.

Reply:
I'll fix it in the refactor PR :)